lib/generation/flutter_project_builder/post_gen_tasks/post_gen_task.dart

```diff
@@ -1,5 +1,16 @@
+import 'package:parabeac_core/controllers/main_info.dart';
+
 /// Abstract class for Tasks that will run post-generation.
 abstract class PostGenTask {
   /// Executes the [PostGenTask].
   void execute();
+
+  // Add elements to analytics
+  void addToAnalytics(String propertyName) {
```

Comment:
This should not go here. I would suggest creating a separate class for analytics or adding the method to a better-suited class 🤔

Reply:
Gotcha, will do!

Reply:
We had done something similar with the Sentry class, maybe we can do what we did for that and register it to `Get it` at the beginning so it's available from anywhere, since analytics may be used from any part of the code 🤔

Reply:
That sounds good to me 👍